examples/with-cookie-auth/pages/api/profile.js

```diff
@@ -17,13 +16,13 @@ const profile = async (req, res) => {
       const js = await response.json()
       // Need camelcase in the frontend
       const data = Object.assign({}, { avatarUrl: js.avatar_url }, js)
-      send(res, 200, { data })
+      return res.status(200).json({ data })
     } else {
-      send(res, response.status, response.statusText)
+      return res.status(response.status).send(response.statusText)
     }
-  } catch (error) {
-    throw createError(error.statusCode, error.statusText)
+  } catch ({ statusText, statusCode }) {
```

Comment:
Where do you get these fields from? I don't see how it works in the node-fetch docs.

Reply:
This implementation is wrong. I'm fixing it. It should be `response.status` and `response.statusText`.

Also, I have to reject the promise if `response.ok` is false, which I'm not doing.